bundles/org.openhab.binding.somfytahoma/src/main/resources/OH-INF/thing/exteriorheatingsystem.xml

```diff
@@ -8,7 +8,8 @@
 		<supported-bridge-type-refs>
 			<bridge-type-ref id="bridge"/>
 		</supported-bridge-type-refs>
-		<label>Somfy Light Switch</label>
+		<label>Somfy Exterior Heating System</label>
```

Comment:
Nice catch! This would require i18n file regerneration and  thing upgrade instructions. Could you split this to a seperate PR?

Reply:
Issue opened.